bundles/org.openhab.binding.emby/src/main/java/org/openhab/binding/emby/internal/handler/EmbyDeviceHandler.java

```diff
@@ -0,0 +1,562 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.emby.internal.handler;
+
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_CONTROL;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_CURRENTTIME;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_DURATION;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_IMAGEURL;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_IMAGEURL_MAXHEIGHT;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_IMAGEURL_MAXWIDTH;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_IMAGEURL_PERCENTPLAYED;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_IMAGEURL_TYPE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_MEDIATYPE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_MUTE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_SHOWTITLE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_STOP;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CHANNEL_TITLE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_GENERALCOMMAND;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_MUTE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_PAUSE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_PLAY;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_SENDPLAY;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_SESSION;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_STOP;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.CONTROL_UNMUTE;
+import static org.openhab.binding.emby.internal.EmbyBindingConstants.DEVICE_ID;
```

Comment:
wildcards are allowed for static imports
```suggestion
import static org.openhab.binding.emby.internal.EmbyBindingConstants.*;
```

Reply:
revised